### DISCUSSION TITLE
destroyOnClose 属性的描述是否叫 销毁节点 更直白一些

### DISCUSSION THREAD

**Discussion by 1587315093** [ant-design/ant-design-mobile#6220] [General]
https://mobile.ant.design/zh/components/tabs

初次看到这个参数以为是Unmount时候执行的回调函数

看到 `不可见时卸载内容` 描述时，又在想了一下卸载内容是啥意思。。。然后看代码才知道是 不可见时直接销毁这个节点

![image](https://github.com/ant-design/ant-design-mobile/assets/77056991/1009ccad-a7f7-4d9c-abcf-bbdd5348057b)

**Comment by IronKinoko** [comment]
![image](https://github.com/ant-design/ant-design-mobile/assets/32838658/d1e39e7b-35fb-4e8d-bcfa-89e9c8038a35)
这图是 antd 的 [Modal](https://ant.design/components/modal-cn#api) 描述

**Reply by 1587315093** [reply]
是的  就是本人觉得 **销毁** 比 **卸载** 更直白，之前第一时间没反应来卸载内容是啥意思

**Reply by IronKinoko** [reply]
![image](https://github.com/ant-design/ant-design-mobile/assets/32838658/a20c82f0-71cf-4be0-8654-bc72c5b27712)
这图是 antd [Tabs](https://ant.design/components/tabs-cn#tabs) 描述
